### ISSUE TITLE
Options for simulated data

### ISSUE THREAD

**Issue by takluyver** [European-XFEL/karabo-bridge-py#41]
Currently, the simulator generates random data for each detector image with `np.random.uniform`. It probably makes sense to have at least an option for sending blank frames (`np.zeros`), which is fast to generate and still useful for testing that tools can receive the data successfully.

We could extend that to simulating things like diffraction rings and crystallography peaks, to allow more complete testing of analysis software.

**Comment by fangohr** [comment]
Yes, fully agree: I like zeros, crystallography peaks (harder), and ring. For imagers put into the beam (I.e. not the detector situation), a 2d gaussian also seems useful (for example like https://github.com/silx-kit/silx/pull/1671/files#diff-817cdc9c507032363f1482705cf4c668R109).

**Comment by takluyver** [comment]
From Sandor's talk today, a couple of programs that could be useful:

- [nonBragg](https://bl831.als.lbl.gov/~jamesh/nonBragg/) simulates scattering from materials like water and ice.
- [MLFSOM](https://bl831.als.lbl.gov/~jamesh/mlfsom/) can take a pdb protein structure and simulate Bragg peaks, including various sources of error.

nonBragg is in C, and MLFSOM in tcsh, so it may not be practical to integrate them directly, but potentially we could generate some fixed samples and overlay them with random noise in Python.
